src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,13 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
-public class HelloWorld {
+import java.util.Random;
 
+public class HelloWorld {
+    public static void main(String[] args) {
+        Figure[] figureS = new Figure[new Random().nextInt(100)];
+        for (int i = 0; i < figureS.length; i++) {
+            figureS[i] = new FigureSupplier().getRandomFigure();
```

Comment:
checklist: You should create several random Figures, so you will use a loop. Please don’t create a new FigureSupplier() inside the loop.